config/locales/en.yml

```diff
@@ -1327,6 +1327,9 @@ in_person_proofing.headings.state_id_milestone_2: Enter the information on your
 in_person_proofing.headings.switch_back: Switch back to your computer to prepare to verify your identity in person
 in_person_proofing.headings.update_address: Update your current address
 in_person_proofing.headings.update_state_id: Update the information on your ID
+in_person_proofing.post_office_closed.body: Post Office locations will resume regular hours on Friday, January 10, 2025.
+in_person_proofing.post_office_closed.heading: All Post Offices will be closed on Thursday, January 9, 2025 to honor former President Jimmy Carter.
```

Comment:
Could we add a &nbsp; so that the name "Jimmy Carter" stays together, rather than splitting up onto two lines?

Reply:
if we do that, let's use the unicode literal version that character of it so we don't have to convert this to an HTML key?